content/test/integration/controller/active-entities.spec.ts

```diff
@@ -272,16 +272,17 @@ loadStandaloneTestEnvironment()('Integration - Get Active Entities', (testEnv) =
       expect(newEntityId).not.toBe(entityId)
       expect(newEntityId).not.toBe('NOT_ACTIVE_ENTITY')
       expect(activeEntities.getCachedEntity('0,1')).toBe(newEntityId)
-
-      const notActiveEntity = activeEntities.getCachedEntity(entityId)
-      expect(notActiveEntity).toBe('NOT_ACTIVE_ENTITY')
+      expect(entityId).toBeDefined()
+      if(entityId) {
```

Comment:
should this code be reached? Or is this just for the linter to work?

Reply:
just for the linter :(